packages/flutter/test/cupertino/switch_test.dart

```diff
@@ -567,7 +567,7 @@ void main() {
     );
 
     expect(find.byType(CupertinoSwitch), findsOneWidget);
-    expect(tester.widget<CupertinoSwitch>(find.byType(CupertinoSwitch)).trackColor, trackColor);
+    expect(tester.widget<CupertinoSwitch>(find.byType(CupertinoSwitch)).inactiveTrackColor, trackColor);
```

Comment:
There should be new tests added. This test makes sure it is still backwards compatible during the deprecation period.